node-installer/script/installer.sh

```diff
@@ -42,6 +42,8 @@ if ! grep -q spin $NODE_ROOT$CONTAINERD_CONF; then
     echo '
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.spin]
     runtime_type = "'$KWASM_DIR'/bin/containerd-shim-spin-v2"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.spin.options]
+    SystemdCgroup = true
```

Comment:
Can we parameterize this in the script by checking if a `CGROUP_DRIVER` env exists and if not make this the default? then, we can add a value to the kwasm chart to configure this:
```sh
helm uninstall \
  kwasm-operator kwasm/kwasm-operator \
  --namespace kwasm \
  --create-namespace \
  --set kwasmOperator.installerImage=ghcr.io/spinkube/containerd-shim-spin/node-installer:v0.18.0
  --set cgroupDriver=systemd
  ```

Reply:
Is `cgroupDriver` needed to be added to kwasm operator?

Reply:
Yes, we'd need to upstream this there. For now, making this configurable with an ENV var with the systemd default is sufficient and then we can add support for the kwasm operator to overwrite that env

Reply:
Cool, once this PR is ready I will create an issue in upstream to track this.